sql/moz-fx-data-shared-prod/fenix_derived/attributable_clients_v2/query.sql

```diff
@@ -0,0 +1,106 @@
+CREATE TEMP FUNCTION sum_map_values(map ARRAY<STRUCT<key STRING, value INT64>>)
+RETURNS INT64 AS (
+  (SELECT SUM(value) FROM UNNEST(map))
+);
+
+WITH client_days AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    sample_id,
+    client_info.client_id,
+    LOGICAL_OR(
+      mozfun.norm.extract_version(client_info.app_display_version, 'major') >= 107
+    ) AS has_search_data,
+    SUM(sum_map_values(metrics.labeled_counter.browser_search_in_content)) AS searches,
+    SUM(sum_map_values(metrics.labeled_counter.browser_search_with_ads)) AS searches_with_ads,
+    SUM(sum_map_values(metrics.labeled_counter.browser_search_ad_clicks)) AS ad_clicks,
+  FROM
+    `moz-fx-data-shared-prod`.fenix.baseline
+  WHERE
+    {% if is_init() %}
+    DATE(submission_timestamp) >= "2021-08-01" {% else %}
+    DATE(submission_timestamp) = @submission_date {% endif %}
```

Comment:
This Jinja formatting by `bqetl format` is unfortunate.  I just pushed #3952 which should improve such Jinja block formatting once it's merged.

Also, with the new Jinja block formatting, consistently trimming trailing spaces after Jinja block statements with `-%}` seems to produce the most readable SQL when rendered IMO.

Reply:
This is great, and looks _much_ better. Thank you @sean-rose!